### DISCUSSION TITLE
Dynamische Liste: Bedeutung und Auswirkung?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2556] [General]
Dynamische Liste
Was passiert bei Auswahl Geräte Label / Geräte Name?
Falls der eine nicht vorhanden, dann nimmt er den anderen.

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/f82531b5-1e6e-4f0e-8f22-88e69a514e5f)
Geräte-Properties -> Welche Eigenschaften?

**Comment by Zefau** [comment]
Die Übersetzung muss ich noch einpflegen.

Die dynamische Liste lädt ja die Geräte anhand der angegebenen Kriterien (z.B. alle Lichter im EG, wenn Gewerk `light` und Etage `EG` angegeben ist und dies in den Geräten so gepflegt ist).

Das Feld Sortierung gibt dann an, in welcher Reihenfolge die dynamisch ermittelten Geräte sortiert werden sollen.

Die Auswahl `Geräte-Label / Geräte-Name` sortiert die Geräte nach Geräte-Label. Da das Feld Label aber optional ist, nutzt jarvis den Geräte-Namen, sofern kein Label angegeben ist. Alternativ kann nur nach Geräte-Namen sortiert werden (zweite Option), dann wird das Label ignoriert.

Die dritte Option die angegeben ist habe ich vergessen zu implementieren. Ziehe ich nach. Hierzu wird es ein Feld im Gerät geben.

**Reply by Zefau** [reply]
https://github.com/Zefau/ioBroker.jarvis/issues/2480

**Reply by mcuiobroker** [reply]
Viele wollen auch die Sortierung nach value haben.
Thema Benzinpreise mehrere Tankstellen.
